frappe/www/login.py

```diff
@@ -17,6 +17,8 @@
 
 def get_context(context):
 	redirect_to = frappe.local.request.args.get("redirect-to")
+	if not redirect_to:
```

Comment:
Should this be after

```
if frappe.session.user != "Guest":
    ...
```

Reply:
@revant can you please explain the benefit of moving the code there?

Reply:
Only logged in users are going to trigger the code block right?

That's why I thought move it there. And similar condition existed there. No other reason.

Reply:
I see. Unless anyone has a strong preference then I think I'll keep it where it is then.
@revant Thanks for your feedback.

Reply:
@surajshetty3416 Any comments?

Reply:
I'll resolve this conversation as it should not be a blocker.